eva_cttv_pipeline/evidence_string_generation/clinvar_to_evidence_strings.py

```diff
@@ -417,12 +388,10 @@ def load_efo_mapping(efo_mapping_file):
                     trait_2_efo[clinvar_name].append((ontology_id, ontology_label))
                 n_efo_mappings += 1
             else:
-                unavailable_efo.add(clinvar_name)
-
-    print(str(n_efo_mappings) + ' EFO mappings loaded')
-    print(str(len(unavailable_efo)) + ' urls without an actual valid EFO mapping')
-
-    return trait_2_efo, unavailable_efo
+                import pdb; pdb.set_trace()
+                print(clinvar_name)
+    logger.info('{} EFO mappings loaded'.format(n_efo_mappings))
+    return trait_2_efo
```

Comment:
Why did you remove the unavailable efo? is this related to adding a column "present in EFO" for the curation?

Reply:
The reasons for removing this test (unavailable in EFO / obsolete) are these two:

> The code does not work, and apparently has not been working for a long time, because it is not syntactically correct (missing matching double quote prevented the curl command from executing).
> Routines for checking whether the terms are in EFO and not obsolete are already implemented properly in other modules, and this check is already being done at the earlier stages of the pipeline.

So, at this point in the pipeline, the check neither _works_ nor _makes sense_ (it's been already implemented at the earlier stages).